README.md

```diff
@@ -50,7 +50,7 @@ The plugin can be configured using the `license` extension on the project.
     ```gradle
     license {
         header = project.file('HEADER.txt')
-        ext {
+        properties {
```

Comment:
Should this be `ext`?

If not, should we try keeping it backward-compatible?

https://github.com/CadixDev/licenser/blob/a52b65c908c4f368042a7e34ae488e4a62e05a93/src/main/groovy/org/cadixdev/gradle/licenser/LicenseExtension.groovy#L268-L271

https://docs.gradle.org/6.8.3/dsl/org.gradle.api.plugins.ExtraPropertiesExtension.html

Reply:
`ext` causes scope issues in kotlin buildscript. It still exists and should continue functioning for groovy buildscripts, but the `properties` method delegates to `ext` and works on both sides. I only documented `properties` in the README to minimize confusion from Kotlin users, since it does work everywhere.